### ISSUE TITLE
author list order should be as provided

### ISSUE THREAD

**Issue by only1chunts** [gigascience/gigadb-website#2130]
## User story

>As a website user
>I want to see the author list as provided by the authors
>So that I have experience that matches the authors input

## Acceptance criteria

>Given I navigate to the gigadb website
>When I am on the dataset page
>Then I can see the list of authors ordered by rank
>And each author name should be of format "First name Middle Name Last Name"
>and each author name is separated by a comma

>Given I navigate to the gigadb website
>When I am on the dataset page
>And I Click "Cite Dataset" and select the "Text" format
>Then I can see the list of authors ordered by rank
>And each author name should be of format "First name Middle Name Last Name"
>and each author name is separated by a comma

>Given I programmatically access the web site
>When I download a XML from the gigadb endpoints
>Then I can see the list of authors ordered by rank
>And each author name should be of format "First name Middle Name Last Name"
>and each author name is separated by a comma

>Given I want information of a dataset files
>When I download the readme file for a dataset
>Then I can see the list of authors ordered by rank
>And each author name should be of format "First name Middle Name Last Name"
>and each author name is separated by a comma

>Given I store metadata about  dataset on DataCite
>When I mint DOI for the dataset
>Then the list of authors sent to Datacite is ordered by rank

## Product Backlog Item Ready Checklist

* [ ] Business value is clearly articulated
* [ ] Item is understood enough by the IT team so it can make an informed decision as to whether it can complete this item
* [ ] Dependencies are identified and no external dependencies would block this item from being completed
* [ ] At the time of the scheduled sprint, the IT team has the appropriate composition to complete this item
* [ ] This item is estimated and small enough to comfortably be completed in one sprint
* [ ] Acceptance criteria are clear and testable
* [ ] Performance criteria, if any, are defined and testable
* [ ] The Scrum team understands how to demonstrate this item at the sprint review

## Product Backlog Item Done Checklist

* [ ] Item(s) in increment pass all Acceptance Criteria
* [ ] Code is refactored to best practices and coding standards
* [ ] Documentation is updated as needed
* [ ] Data security has not been compromised (with particular reference to the personal information we hold in GigaDB)
* [ ] No deviation from the team technology stack and software architecture has been introduced
* [ ] The product is in a releasable state (i.e. the increment has not broken anything)

**Comment by alli83** [comment]
@only1chunts at the moment, it's separated by a semicolon, do you want to update it?

Given I navigate to the gigadb website
When I am on the dataset page
Then I can see the list of authors ordered by rank
And each author name should be of format "First name Middle Name Last Name"
and each author name is separated by a comma

**Comment by only1chunts** [comment]
yes, the acceptance criteria are written correct. We wish to align with the
way the journal displays authors names on their website.

**Comment by rija** [comment]
Hi @alli83, 

I can't see a PR connected to this ticket, but I seem to recall you have done this work.
Could you confirm?

**Comment by rija** [comment]
A PR is ready for review (#2133), so closing

**Comment by only1chunts** [comment]
FYI- this image shows how the authors are displayed in GigaByte journal articles, and its what we're aiming to replicate in GigaDB:

![Image](https://github.com/user-attachments/assets/2d026bb9-5cee-4da8-8b3f-16c977a8ba2e)

I think the show more button is probably not needed in GigaDB, but if we find that datasets with lots of authors become unsightly we can create a new ticket for that.
